cmake/OpenCVModule.cmake

```diff
@@ -936,15 +958,30 @@ function(ocv_add_perf_tests)
       add_dependencies(opencv_perf_tests ${the_target})
 
       # Additional target properties
-      set_target_properties(${the_target} PROPERTIES
-        DEBUG_POSTFIX "${OPENCV_DEBUG_POSTFIX}"
-        RUNTIME_OUTPUT_DIRECTORY "${EXECUTABLE_OUTPUT_PATH}"
-      )
-
+      if(NOT WINRT)
+        set_target_properties(${the_target} PROPERTIES
+          DEBUG_POSTFIX "${OPENCV_DEBUG_POSTFIX}"
+          RUNTIME_OUTPUT_DIRECTORY "${EXECUTABLE_OUTPUT_PATH}"
+        )
+      else()
+        # WINRT has issues in case name in appxManifest and real executable name mismatch
+        set_target_properties(${the_target} PROPERTIES
+          DEBUG_POSTFIX ""
+          RUNTIME_OUTPUT_DIRECTORY "${EXECUTABLE_OUTPUT_PATH}"
+        )
+      endif()
```

Comment:
Updated this. Please note that empty `OPENCV_DEBUG_POSTFIX` is not required and shouldn't be set for modules to keep possibility to build and install them in Debug/Release both. This modification is for tests only so the flag is not set in the global place.